### PR TITLE
clang-compiler-activation v13.0.1

### PR DESCRIPTION
**Pull request by h-vetinari**
Continuing from https://github.com/conda-forge/clang-compiler-activation-feedstock/pull/69, since the version bot had an [error](https://conda-forge.org/status/#version_updates):

<details>

```
The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!

Please check the URLs in your recipe with version '13.0.1' to make sure they exist!

We also found the following errors:

 - migrations do not work on versions not specified with jinja2
```

</details>

I chose to add 13.0.1 without dropping 11.1 or 12.0, but that would be easily doable, of course. Also added myself to maintainers following [this](https://github.com/conda-forge/clang-compiler-activation-feedstock/pull/69#issuecomment-933036007) comment.

### CODE REVIEW

**Comment by h-vetinari** [comment]
Blockers (indentation denotes dependency; c.f. also list from [12.0.1](https://github.com/conda-forge/clang-compiler-activation-feedstock/pull/63#issuecomment-877005303)):
* [ ] https://github.com/conda-forge/llvm-meta-feedstock/pull/13
* [x] https://github.com/conda-forge/llvmdev-feedstock/pull/140
  * [ ] https://github.com/conda-forge/openmp-feedstock/pull/57
  * [ ] https://github.com/conda-forge/lld-feedstock/pull/33
  * [ ] https://github.com/conda-forge/clangdev-feedstock/pull/163
    * [ ] https://github.com/conda-forge/libcxx-feedstock/pull/83
    * [ ] update https://github.com/conda-forge/cctools-and-ld64-feedstock
    * [ ] https://github.com/conda-forge/compiler-rt-feedstock/pull/37

**Comment by h-vetinari** [comment]
@conda-forge-admin, please rerender

**Comment by h-vetinari** [comment]
@conda-forge-admin, please rerender

**Comment by h-vetinari** [comment]
@isuruf 
This should be the final piece to land LLVM 13 - I used the same list of dependencies as for 12 & 11, so I don't think anything should be missing here. I propose decoupling the build-numbers for the different LLVM versions, see the last commit (but no strong opposition to keep incrementing globally either).

**Comment by h-vetinari** [comment]
Gentle ping @isuruf

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -34,3 +34,4 @@ zip_keys:
 version:
   - 11.1.0
```
Comment (isuruf) [inline recipe/conda_build_config.yaml:35]: Let's drop 11.1.0

Comment (h-vetinari) [reply recipe/conda_build_config.yaml]: Done!

**Comment by isuruf** [comment]
Thanks

**Comment by h-vetinari** [comment]
Woohoo, finally LLVM 13 is available! 🥳

**Comment by h-vetinari** [comment]
LLVM 14 should be a bunch closer to the upstream release, I'm hoping (most of the blocking feedstocks have LLVM 14 rc's building successfully already). 🙃

**Pull request merged**